### PR TITLE
Backport to 102X: Ecal Bad calib filter adding a new problematic crystal

### PR DESCRIPTION
**Pull request by eioannou**
Backport of #24878

### CODE REVIEW

File: RecoMET/METFilters/python/ecalBadCalibFilter_cfi.py [hunk 1]
```diff
@@ -29,4 +29,4 @@
                                                        872423215,872416066,872435036,872439336,
                                                        872420273,872436907,872420147,872439731,
                                                        872436657,872420397,872439732,872439339,
-                                                       872439603])
+                                                       872439603,872422436])
```
Comment (slava77) [inline RecoMET/METFilters/python/ecalBadCalibFilter_cfi.py]: the same comment as in https://github.com/cms-sw/cmssw/pull/25186/files#r232349972 applies here

**Comment by slava77** [comment]
@cmsbuild please test

File: RecoMET/METFilters/python/ecalBadCalibFilter_cfi.py (hunk 1 above)
Comment (ahinzmann) [reply RecoMET/METFilters/python/ecalBadCalibFilter_cfi.py]: Until plans for a new MiniAOD in 102X and a new era is defined, this backport may stay on the todo list (hold?) then.

**Comment by slava77** [comment]
-1

To enter production release this PR will need to be updated to pick up the needed change only for the (future) production process/era configuration modifier.
We can keep track of this in #25365

**Comment by peruzzim** [comment]
This is superseded by #25459

**Comment by davidlange6** [comment]
Your PR is unmergeable. Please have a look and possibly rebase it.

**Comment by fabiocos** [comment]
@eioannou if this PR has been replaced by something else, please close it

**Comment by eioannou** [comment]
@fabiocos Done.

**Pull request closed without merging**